app/test.jsx

```diff
@@ -115,26 +125,37 @@ class App extends React.Component {
             high,
             low,
         }] : [];
+
+        const MarkerTimes = [];
+
+        for (let i = 0; i < 5; i++) {
+            MarkerTimes.push(moment().utc().second(0).subtract(i + 3, 'minutes')
+                .unix());
+        }
+
         return (
             <div className="grid">
                 <div className="chart-instance">
                     <SmartChart
                         onSymbolChange={symbol => console.log('Symbol has changed to:', symbol)}
                         isMobile={isMobile}
-                        chartControlsWidgets={renderControls}
+                        chartControlsWidgets={this.renderControls}
                         requestAPI={requestAPI}
                         requestSubscribe={requestSubscribe}
                         requestForget={requestForget}
                         barriers={barriers}
-                    />
-                </div>
-                <div className="side-chart">
-                    <SmartChart
-                        isMobile={isMobile}
-                        requestAPI={requestAPI}
-                        requestSubscribe={requestSubscribe}
-                        requestForget={requestForget}
-                    />
+                        granularity={this.state.granularity}
+                    >
+                        {MarkerTimes.map(x => (
+                            <Marker
+                                key={x}
+                                className="chart-marker-line"
+                                x={x}
+                                xPositioner="epoch"
+                                yPositioner="top"
```

Comment:
@shayanbinary IMO, we can handle the `yPositioner` to `top` also in `MarkerStore` after adding the invisible bar, because the third-party applications don't care about it that the `x` exists exactly in the `x-axis` or not.